src/pages/wallet/CreateWalletTopUp.tsx

```diff
@@ -247,8 +247,8 @@ const CreateWalletTopUp = () => {
   }, [formikProps.dirty, navigateBack])
 
   const hasMinMax =
-    typeof wallet?.paidTopUpMinAmountCents !== 'undefined' ||
-    typeof wallet?.paidTopUpMaxAmountCents !== 'undefined'
+    (wallet?.paidTopUpMinAmountCents !== null && wallet?.paidTopUpMinAmountCents !== undefined) ||
```

Comment:
Is it maybe time to create a core function `isEmpty` where we test those kind of scenarios? I feel like it's not the first time we stumble across those and this way, you can use this excuse to test empty strings and everything